packages/mint-components/src/components/tax-and-cash/sqm-banking-info-form/sqm-banking-info-form.feature

```diff
@@ -0,0 +1,107 @@
+@owner:zach @author:zach
+Feature: Banking Information Form
+
+  Background: A participant has submitted a tax form and would like to fill out their banking information
+
+  Scenario: Loading skeletons are shown when the form is loading
+    When the participant loads the form
+    And they have not selected a payout method
+    And the form is in a loading state
+    Then the form shows a loading skeleton in place of the payment method checkboxes
+    But if they have selected a payment method
+    Then the form shows skeletons for the expected number of input fields
+
+  Scenario: A banner is shown at the top of the page if the partner already exists
```

Comment:
Is this still the case @Locrian24 or is the behaviour a bit updated now?

Reply:
The banner is shown in the case where they are a non-existing brand partner. I should be hidden if they came to the step via the "Edit Payout Information" button on the Dashboard.